src/components/Annotation/Comment/EditableComment.tsx

```diff
@@ -46,46 +50,78 @@ export const EditableComment = (props: EditableCommentProps) => {
     }
   }, [editable]);
 
+  useEffect(() => {
+    if (comment.value && comment.value.length > 0) {
+      if (comment.value.charAt(0) === '{') {
+        setRenderType('quill');
+      } else {
+        setRenderType('text');
+      }
+    }
+  }, []);
+
   const onSaveChange = (evt: React.FormEvent) => {
     evt.preventDefault();
 
     store.updateBody(comment, {
       ...comment,
-      value: textarea.current!.value
+      render_type: renderType,
```

Comment:
Yep, I think this should simply be replaced by `format` and then either with a custom value (`plaintext` vs. `quill` or `text/plain` vs. `application/json`.

Reply:
So you are saying replace render_type with format? And then defining a new format for quill? Where would I do that?

Reply:
My thought would be, simply, to really just replace `render_type` with `format`. No other change needed. `format` is already a defined field in the spec. And then we'd just have `text/plain` and `quill` as the two possible values of `format`. Would be a minimal change, but has the advantage of making annotations fully spec compliant, if they are plaintext; and still pretty much spec-compliant for Quill-annotations. ("Pretty much" in the sense that if we export the annotations, other WebAnno clients would be able to look up the `format` field, and decide that they can't just render it as plaintext.)

Reply:
I will replace with format.

Reply:
So I have replace with format but that is not be persisted to the DB.  Can you take a look at that?

Reply:
Oh - good point. Come to think of it: I don't think `render_type` ever got persisted, either right? It looks like the decision in the client is based by inspecting the value, and checking if the first char is a `{`. 

Here's my suggestion:
- Good news is: the DB `bodies` table already have a `format` field. (We added that way back in the beginning but never used it.) The permitted values currently are `TextPlain` and `TextHtml`. Can you change that to `text/plain` and `text/quill`?
- Then I'll make sure that `format` actually gets stored. (This is a small fix in the [Annotorious/Supabase storage plugin](https://github.com/recogito/annotorious-supabase).)

Reply:
__Update:__ I now made the changes to the Annotorious Supabase plugin. (See [here](https://github.com/recogito/annotorious-supabase/commit/1f1b683d4d28ce131c5cce2421d871a5ab15b2f8).) I updated the version in the `package.json`. I.e. if you run `git pul`, `npm install`, the format field should get persisted.